pkg/services/object/get/util.go

```diff
@@ -368,6 +357,47 @@ func (c *clientCacheWrapper) InitGetObjectStream(ctx context.Context, node netma
 	return hdr, rc, nil
 }
 
+func (c *clientCacheWrapper) Head(ctx context.Context, node netmap.NodeInfo, pk ecdsa.PrivateKey, cnr cid.ID, id oid.ID,
+	sTok *session.Object) (object.Object, error) {
```

Comment:
No bearer? It seems to be a generic Head, not EC-specific one. It should support bearer, but then EC can never pass it.

Reply:
I was sure it's there in `InitGetObjectStream`, but seems like it's not. OK, let's not touch this for now. Wrapped wrappers of wrappers in another issue.